project-base/composer.json

```diff
@@ -50,6 +50,7 @@
         "doctrine/doctrine-fixtures-bundle": "^3.0.2",
         "doctrine/doctrine-migrations-bundle": "^1.3.0",
         "egeloen/ckeditor-bundle": "^4.0.6",
+        "elasticsearch/elasticsearch": "^6.0.1",
```

Comment:
Shouldnt this be rather framework's dependency?

Reply:
You are right, I have moved it